site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,247 @@
++++
+date = "2018-05-25T13:00:00+05:30"
+title = "Citra Progress Report 2018 Q1"
+tags = [ "progress-report" ]
+author = "CaptV0rt3x"
+forum = 0000
++++
+
+It's been a while folks. Our content writer [***Anodium***](https://github.com/anodium)
+has had to take a break to deal with the aftermath of hurricane Maria. So, we
+haven't had anyone to write up a progress report till now.
+
+Coming to new changes in Citra since the last report, we have had many new features
+added in Citra. Some notable ones are multiplayer, hardware shader improvements (post GLvtx),
+logging rewrite and the highly coveted camera support. Our build generating environment
+was vastly improved as well. Apart from these, we have many more minor features,
+improvements and bug fixes.
+
+***Now let’s see these in detail:***
+
+## Citra-qt : Multiplayer support by [jroweboy](https://github.com/jroweboy), [B3N30](https://github.com/B3N30), [Subv](https://github.com/subv) and [JayFoxRox](https://github.com/jayfoxrox)
+
+Citra has been able to run many games at playable speeds for a while now. But it
+was always lacking something which the 3DS had. The ability to play with friends.
+So, to make it a reality our developers [jroweboy](https://github.com/jroweboy),
+[B3N30](https://github.com/B3N30), [Subv](https://github.com/subv) and
+[JayFoxRox](https://github.com/jayfoxrox) worked tirelessly for months to reverse
+engineer the working of 3DS local wireless system. Citra doesn't support or emulate
+online play or download play (***yet!*** ). But what it does, is that it now emulates
+_local wireless multiplayer_ over the internet. This allows you to play your most
+favorite games with your virtual friends across the world.
+
+**Note**: Although multiplayer is supported, some games might have known/unknown issues.
```

Comment:
>some games might have known/unknown issues

some games might experience issues
it's a little bit silly to put known and unknown there

Reply:
fixed